amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -297,10 +299,13 @@ export class DashboardPage extends React.Component<
                         : NO_TIMESTAMP_TEXT}
                     </time>
                     <div className="last-run-state">
-                      <Flag
-                        caseType="sentenceCase"
-                        text={dashboard.last_run_state}
-                        labelStyle={this.mapStatusToStyle(
+                      <ResourceStatusMarker
```

Comment:
We still want to make sure this shows the same text. `Succeeded` or `Failed`, whereas it's the upcoming SLA use case on tabledeatil page would pass in `Hit` & `Missed`.